x-pack/plugin/ml-feature-index-builder/src/main/java/org/elasticsearch/xpack/ml/featureindexbuilder/persistence/DataFramePersistentTaskUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.ml.featureindexbuilder.persistence;
+
+import org.elasticsearch.cluster.ClusterState;
+import org.elasticsearch.cluster.metadata.MetaData;
+import org.elasticsearch.persistent.PersistentTasksCustomMetaData;
+import org.elasticsearch.xpack.ml.featureindexbuilder.FeatureIndexBuilder;
+
+public final class DataFramePersistentTaskUtils {
+
+    private DataFramePersistentTaskUtils() {
+    }
+
+    /**
+     * Check to see if the PersistentTask's cluster state contains the job(s) we
+     * are interested in
+     */
+    public static boolean stateHasDataFrameJobs(String id, ClusterState state) {
+        boolean hasJobs = false;
+        PersistentTasksCustomMetaData pTasksMeta = state.getMetaData().custom(PersistentTasksCustomMetaData.TYPE);
+
+        if (pTasksMeta != null) {
+            // If the request was for _all jobs, we need to look through the
+            // list of
```

Comment:
I think you should be able to squash this comment in 2 lines

Reply:
Eclipse formatting drives me crazy! -> https://github.com/elastic/elasticsearch/pull/35984